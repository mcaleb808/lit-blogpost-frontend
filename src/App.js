import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import Routes from "./components/Routes";
import "bootstrap/dist/css/bootstrap.min.css";

const App = () => (
  <Router>
    <Routes />
  </Router>
);

export default App;
