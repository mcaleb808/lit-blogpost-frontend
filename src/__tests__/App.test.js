import React from "react";
import { shallow } from "enzyme";
import App from "../App";

describe("<App />", () => {
  test("should render the <App />", () => {
    let wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
