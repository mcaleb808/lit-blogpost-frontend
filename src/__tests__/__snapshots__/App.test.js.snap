// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render the <App /> 1`] = `ShallowWrapper {}`;
